.github/workflows/markdown-lint-fix.yml

```
@@ -0,0 +1,46 @@
+name: Create Markdownlint auto-fix PR
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  fix:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+
+      - name: Install all yarn packages
+        run: |
+          yarn --frozen-lockfile
+
+      - name: Lint markdown files
+        run: |
+          yarn lint:md
+
+      - name: Create PR with only fixable issues
+        if: success()
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: "chore: auto-fix Markdownlint issues"
+          branch: daily-markdownlint-fix
+          title: "Daily Markdownlint auto-cleanup"
```

```suggestion
          title: "Daily Markdownlint auto-cleanup"
          author: mdn-bot <108879845+mdn-bot@users.noreply.github.com>
```
Makes the @mdn-bot the author.